File: OPENAPI-REST-API/swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/DomainAdminContact.scala
package io.swagger.server.model


/**
 * Administrative contact information for a domain registration.
 *
 * @param status 
 * @param state 
 * @param org_name 
 * @param country 
 * @param postal_code 
 * @param email 
 * @param fax 
 * @param address2 
 * @param address3 
 * @param address1 
 * @param city 
 * @param phone 
 * @param first_name 
 * @param last_name 
 */
case class DomainAdminContact (
  status: Option[String],
  state: Option[String],
  org_name: Option[String],
  country: Option[String],
  postal_code: Option[String],
  email: Option[String],
  fax: Option[String],
  address2: Option[String],
  address3: Option[String],
  address1: Option[String],
  city: Option[String],
  phone: Option[String],
  first_name: Option[String],
  last_name: Option[String]
)

File: OPENAPI-REST-API/openapi-client/php-dt/src/App/DTO/DomainAdminContact.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\PhpAttribute as DTA;

/**
 * Administrative contact information for a domain registration.
 */
class DomainAdminContact
{
    #[DTA\Data(field: "status", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $status = null;

    #[DTA\Data(field: "state", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $state = null;

    #[DTA\Data(field: "org_name", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $org_name = null;

    #[DTA\Data(field: "country", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $country = null;

    #[DTA\Data(field: "postal_code", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $postal_code = null;

    #[DTA\Data(field: "email", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $email = null;

    #[DTA\Data(field: "fax", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $fax = null;

    #[DTA\Data(field: "address2", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $address2 = null;

    #[DTA\Data(field: "address3", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $address3 = null;

    #[DTA\Data(field: "address1", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $address1 = null;

    #[DTA\Data(field: "city", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $city = null;

    #[DTA\Data(field: "phone", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $phone = null;

    #[DTA\Data(field: "first_name", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $first_name = null;

    #[DTA\Data(field: "last_name", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $last_name = null;

}
